{{cookiecutter.repo_name}}/setup.py

```diff
@@ -169,9 +171,16 @@ def read(*names, **kwargs):
         'Operating System :: POSIX',
         'Operating System :: Microsoft :: Windows',
         'Programming Language :: Python',
+{%- if cookiecutter.legacy_python == "yes" %}
         'Programming Language :: Python :: 2.7',
+{%- endif %}
         'Programming Language :: Python :: 3',
+{%- if cookiecutter.legacy_python == "no" %}
+        # 'Programming Language :: Python :: 3 :: Only',
```

Comment:
We can uncomment this, it's not like the user could install it on python2 anyway with the python_requires='>=3.6' constraint.

Reply:
I was thinking about Python 4.

_python_requires could also be `~=3.6` I think to indicate not Python 4. But I would have to look again to be sure._

Reply:
Live in the now, not in the whatcouldbe I'd say :-) Make things consistent.

Reply:
Ok.